README.md

```diff
@@ -1,31 +1,43 @@
-# nornir_nautobot
 
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+<div style="text-align: center;"> 
+  <h1>Nornir Nautobot</h1>
+</div>  
+<p align="center">
+  <a href="https://github.com/nautobot/nornir-nautobot/actions"><img src="https://github.com/nautobot/nornir-nautobot/actions/workflows/ci.yml/badge.svg?branch=main"></a>
+  <a href="https://pypi.org/project/nornir-nautobot/"><img src="https://img.shields.io/pypi/v/nornir-nautobot"></a>
+  <a href="https://pypi.org/project/nornir-nautobot/"><img src="https://img.shields.io/pypi/dm/nornir-nautobot"></a>
+  <br>
+</p>
 
```

Comment:
Would prefer to see this in Markdown rather than HTML. Are we able to convert that over?

Reply:
I'm guessing this comes from the nautobot-plugin docs template. Since there's no logo or custom styling, you could convert to md links. The center align doesn't have an equivalent.

Reply:
It came from another plugin repo, yes. Used html because I wanted to center it (like the plugin repos), and the links to github, etc looked strange on their own in the middle without an app logo. Regardless, I've changed them all to markdown and it looks fine as well that way.